pkg/apis/crd/v1alpha2/types.go

```diff
@@ -186,6 +186,10 @@ type AppliedTo struct {
 	// Groups is the set of ClusterGroup names.
 	// +optional
 	Groups []string `json:"groups,omitempty"`
+	// Select all Pods with the ServiceAccount matched by this field.
+	// Cannot be set with any other selector.
+	// +optional
+	ServiceAccounts []v1alpha1.ServiceAccount `json:"serviceAccounts,omitempty"`
```

Comment:
The `AppliedTo `struct is used by `Egress ` only today (though I do hope we change NetworkPolicy and Rule to use `AppliedTo` for the AppliedTo field). Do you plan to support Egress in this PR? If not, no need to change the struct.

Reply:
Make sense. Removed.